src/js/constants.js

```diff
@@ -46,12 +46,15 @@ export const MediaStateChangeEvents = {
   MEDIA_SEEKABLE: 'mediaseekablechange',
   MEDIA_PREVIEW_IMAGE: 'mediapreviewimagechange',
   MEDIA_PREVIEW_COORDS: 'mediapreviewcoordschange',
-  // MEDIA_CHROME_ATTRIBUTES: 'media-chrome-attributes',
-  // MEDIA_CONTROLLER: 'media-controller',
   MEDIA_LOADING: 'medialoadingchange',
   USER_INACTIVE: 'userinactivechange',
 };
 
+export const UIControllerAttributes = {
+  MEDIA_CHROME_ATTRIBUTES: 'media-chrome-attributes',
+  MEDIA_CONTROLLER: 'media-controller',
+};
+
 export const MediaUIAttributes = {
```

Comment:
I was tempted to rename MediaUIAttributes to MediaStateAttributes to be clearer, but that would have touched a lot of files and it's only an internal thing.

Reply:
I think that would be a good followup renaming though.